caffeinated/modules/recaptcha_invisible/assets/espresso_invisible_recaptcha.js

```diff
@@ -24,6 +24,16 @@ function espressoLoadRecaptcha() {
         if (typeof eeRecaptcha === 'undefined'){
             return;
         }
+
+        // Display an error.
+        var display_error = function (msg) {
+            $('#espresso-ajax-notices').eeCenter('fixed');
+            var espresso_ajax_msg = $('#espresso-ajax-notices-error');
+            espresso_ajax_msg.children('.espresso-notices-msg').html(msg);
+            $('#espresso-ajax-loading').fadeOut('fast');
+            espresso_ajax_msg.removeClass('hidden').show().delay(5000).fadeOut();
```

Comment:
I think we should display the notice longer than 5 seconds, maybe 8-10 seconds, since 5 seconds makes it too easy to miss, if you happen to look away from your screen for a bit.
I can't remember off the top of my head , but does the modal window the error is appearing in have a button for closing it?
If so, then let's definitely bump the time for the message up to like 12-15 seconds since ppl would be able to close it.

Reply:
Yeah, I agree. Thanks.